src/oom/oomd-util.c

```diff
@@ -278,6 +284,25 @@ int oomd_cgroup_context_acquire(const char *path, OomdCGroupContext **ret) {
         if (r < 0)
                 return log_debug_errno(r, "Error parsing memory pressure from %s: %m", p);
 
+        r = cg_get_owner(SYSTEMD_CGROUP_CONTROLLER, path, &uid, &gid);
+        if (r < 0)
+                log_debug_errno(r, "Failed to get owner/group from %s: %m", path);
+        else if (uid == 0 && gid == 0) {
```

Comment:
hmm, is there any good reason to check gid == 0 here? group doesn't really matter, does it? uid == 0 matters, but the group?

Reply:
yea i guess the group doesn't matter here